### PR TITLE
docs: Fix policy-ref formatting, GraphQL docs

### PR DESCRIPTION
**Pull request by philipaconrad**
This commit fixes the formatting of "note" fields in the policy reference document that were block quotes before, and adds a note about supporting custom GraphQL `@directive` syntax in schemas.

It also corrects the GraphQL tutorial source code link, over in the GraphQL API authz tutorial.

### CODE REVIEW

File: docs/content/policy-reference.md [hunk 1]
```diff
@@ -724,6 +726,14 @@ all_paths[entity_name]
 
 {{< builtin-table cat=graphql title="GraphQL" >}}
 
+{{< info >}}
+Custom [GraphQL `@directive`][graphql-spec-directives] definitions defined by your GraphQL framework will need to be included manually as part of your GraphQL schema string in order for validation to work correctly on GraphQL queries using those directives.
+A common way to add these directive definitions if they are defined separately from your schema is to `concat` them to your schema string right before use.
+Directives defined as part of the GraphQL specification (`@skip`, `@include`, `@deprecated`, and `@specifiedBy`) are supported by default, and do not need to be added to your schema manually.
+{{< /info >}}
+
+   [graphql-spec-directives]: http://spec.graphql.org/October2021/#sec-Language.Directives
```
Comment (srenatus) [inline docs/content/policy-reference.md:735]: [nit] A little uncommon, this reference style. But it doesn't hurt, I suppose.

File: docs/content/graphql-api-authorization.md [hunk 2]
```diff
@@ -519,5 +519,5 @@ You learned a number of things about API authorization with OPA:
 * You can use a remote bundle server for distributing policy and data.
 
 The code for this tutorial can be found in the
-[open-policy-agent/contrib](https://github.com/open-policy-agent/contrib)
+[StyraInc/graphql-apollo-example][graphql-example-repo]
```
Comment (srenatus) [inline docs/content/graphql-api-authorization.md:522]: 👍 Let's add the repo to integrations.yml, too.

**Review by srenatus — APPROVED** [review]
nitpicks/questions only

File: docs/content/policy-reference.md (hunk 1 above)
Comment (philipaconrad) [reply docs/content/policy-reference.md]: For the `concat` bit, would a separate example block be worthwhile, you think?

File: docs/content/policy-reference.md [hunk 3]
```diff
@@ -724,6 +726,14 @@ all_paths[entity_name]
 
 {{< builtin-table cat=graphql title="GraphQL" >}}
 
+{{< info >}}
+Custom [GraphQL `@directive`][graphql-spec-directives] definitions defined by your GraphQL framework will need to be included manually as part of your GraphQL schema string in order for validation to work correctly on GraphQL queries using those directives.
+A common way to add these directive definitions if they are defined separately from your schema is to `concat` them to your schema string right before use.
```
[the remark this answers is not in the document]
Comment (philipaconrad) [reply docs/content/policy-reference.md:731]: It's a good question. I meant the builtin `concat`. I'll see about adding an example, separate from that info block.

File: docs/content/policy-reference.md (hunk 1 above)
Comment (philipaconrad) [reply docs/content/policy-reference.md]: Also, the reference style doesn't seem to be supported in `{{< info >}}` blocks, so I'm going to try the direct-link style, and if that fails, just paste the raw link in there. :sweat_smile:

Comment (philipaconrad) [reply docs/content/policy-reference.md]: Direct-link (`[text](link)`) style seems to be supported. :+1:

File: docs/content/graphql-api-authorization.md (hunk 2 above)
Comment (philipaconrad) [reply docs/content/graphql-api-authorization.md:522]: It's not a proper integration, just a tutorial. Does that still work for `integrations.yml`?

Comment (srenatus) [reply docs/content/graphql-api-authorization.md:522]: I thought about adding the repo [here](https://github.com/open-policy-agent/opa/blob/main/docs/website/data/integrations.yaml#L22)

File: docs/content/policy-reference.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (philipaconrad) [reply docs/content/policy-reference.md]: It is done. We have an example for `concat`'ing custom directives. It took me a bit to wrangle the GraphQL syntax juuuuust right, but it's a representative example, I think. :smiley:

**Review by srenatus — APPROVED** [review]
LGTM! Thanks.

**Comment by srenatus** [comment]
Perhaps we can combine the two graphql related tiles in the integration page...

**Pull request merged**